### PR TITLE
charts/kube-prometheus-stack: fix typos

### PR DESCRIPTION
**Pull request by invidian**
Splitted #313 per chart as requested.

Signed-off-by: Mateusz Gozdek <mgozdekof@gmail.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml [hunk 1]
```diff
@@ -159,7 +159,7 @@ data:
                         "dashes": false,
                         "datasource": "$datasource",
                         "decimals": 3,
-                        "description": "How much error budget is left looking at our 0.990% availability gurantees?",
+                        "description": "How much error budget is left looking at our 0.990% availability guarantees?",
```
Comment (bismarck) [inline charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml:162]: Please see the note at the top of this file.

**Review by bismarck — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml (hunk 1 above)
Comment (invidian) [reply charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml:162]: Oh, let me create a PR upstream and remove this. Thanks.

Comment (invidian) [reply charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml:162]: https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/513

Comment (invidian) [reply charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml]: Removed.

**Review by monotek — APPROVED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
please raise chart version to 11.0.4

**Comment by invidian** [comment]
>please raise chart version to 11.0.4

Done :+1:

**Review by monotek — APPROVED** [review]

**Review by gkarthiks — DISMISSED** [review]

**Review by bismarck — APPROVED** [review]
lgtm

**Pull request merged**